woocommerce/woocommerce-ios
# [Blaze] Intro view for campaign creation

Part of: #11565


## Description

Adds the new intro view from Blaze i3 design. 

We are presenting this new intro view instead of the old one after checking for the feature flag. 

## Testing instructions
- Create a JN site
- Create a new product and publish it

Dashboard
- From the Dashboard Blaze view tap "Create Campaign" button
- You should see the new intro view

Product form
- Open a published product 
- Tap on "Promote with Blaze" button
- You should see the new intro view

Blaze settings
- Open Menu -> Blaze
- You should see the new intro view

Feature flag off
- Turn off the feature flag `blazei3NativeCampaignCreation` and follow the above steps and ensure that old intro view is displayed.

## Screenshots

| Light | Dark |
|--------|--------|
| ![Simulator Screenshot - iPhone 14 Pro - 2023-12-27 at 21 07 19](https://github.com/woocommerce/woocommerce-ios/assets/524475/29f1c423-c9f6-432b-a059-c1e575b0356f) | ![Simulator Screenshot - iPhone 14 Pro - 2023-12-27 at 21 07 28](https://github.com/woocommerce/woocommerce-ios/assets/524475/4c7f3aeb-d7d8-49cc-8e43-0fe77410576c) | 

---
